sct.py

```diff
@@ -1668,6 +1672,67 @@ def configure_jenkins_builders(cloud_provider, regions):
             raise NotImplementedError("configure_jenkins_builders doesn't support Azure yet")
 
 
+@cli.command("create-argus-test-run", help="Initialize an argus test run.")
+def create_test_run():
+    try:
+        argus_test_id_path = Path("argus_test_id")
```

Comment:
We have possibility to run 2+ different tests on a single SCT runner:
1) Performance throughput CI job defined at `jenkins-pipelines/operator/performance/perf-regression-throughput-eks.jenkinsfile` (3 parallel tests).
2) CI tests which we can trigger with labels in `github`. Example: https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6162/9/console (5 parallel tests)

As a result, if I understand the new logic correctly then it may cause mess in the above mentioned cases.

So, need to use a `test logdir` the same way as the `TestConfig` does it storing there it's `test_id` file.

Reply:
Regarding sharing of runners, all those examples are not sharing runners, and they do we need to fix that.

I do agree we should use the test dir, the tricky part, is that test dir is created when the test starts, so you can put anything in it before that, you'll need something equivalent, that can be found at later stages while not assuming only one test is running at a runner.